x-pack/plugin/autoscaling/src/main/java/org/elasticsearch/xpack/autoscaling/Autoscaling.java

```diff
@@ -79,6 +81,12 @@
         }
     }
 
+    static final LicensedFeature.Momentary AUTOSCALING_FEATURE = LicensedFeature.momentary(
+        null,
+        "autoscaling",
+        License.OperationMode.ENTERPRISE
+    );
```

Comment:
Question: Do we prefer `null` over reusing the feature name as its own family? I have no strong view. Just curious.

Reply:
Yes null means no family. Family is just a way to group several similar features, but there isn't any grouping that I know of for autoscaling.